models/test.py

```diff
@@ -0,0 +1,91 @@
+import sys, os
+parent_dir = os.getcwd()
+sys.path.append(parent_dir)
+import tensorflow as tf
+import keras
+from keras.models import Sequential
+from keras.applications.vgg16 import VGG16
+from keras.models import Model
+from keras.layers import Input, Flatten, Dense, Dropout
+from keras.optimizers import SGD
+from keras.backend import eval
+import numpy as np
+from optimizer.LR_SGD import LR_SGD
+from OurModel import OurModel
+from keras.legacy import interfaces
+from DataLoader import DataLoader
+from Trainer import Trainer
+
+filepath = 'C:\\Users\\ckrem\\Desktop\\IP\\age.tfrecords '
```

Comment:
wieso keine relativen pfade?
```python
os.getcwd() # returns the current path.
```

Reply:
It was just for testing issues. We haven't had specified a structure where to put the tfrecords at that moment. Thanks for the advice with os.getcwd() :)